jazzmin/templates/admin/base.html

```diff
@@ -99,6 +99,24 @@
                 </li>
                 {% endif %}
 
+                {% if jazzmin_settings.language_chooser %}
+                    {% get_available_languages as LANGUAGES %}
+                    {% get_language_info_list for LANGUAGES as languages %}
+
+                    <li class="nav-item dropdown">
+                        <a class="nav-link btn" data-toggle="dropdown" href="#" title="Choose language">
+                            <i class="fas fa-globe-americas pr-2" aria-hidden="true"></i>
```

Comment:
I would rather use `<i class="fas fa-flag"></i>` here

Reply:
Unfortunately, the flag does not look like a language icon, more of a racecar icon, ive changed it to `fas fa-globe` so its not biased, though that icon did look most recognisable as a language setting icon